fstat/controller.py

```diff
@@ -26,7 +27,7 @@ def index():
 
 @app.route('/login', methods=['POST'])
 def login():
-    return github.authorize(scope="user:email,read:org")
+    return github.authorize(scope="user,user:email,read:org")
```

Comment:
Have you checked if the profile picture bits work with just the email permissions?